src/control/server/storage/scm/ipmctl_dimm.go

```diff
@@ -0,0 +1,169 @@
+//
+// (C) Copyright 2022-2024 Intel Corporation.
+//
+// SPDX-License-Identifier: BSD-2-Clause-Patent
+//
+
+package scm
+
+import (
+	"encoding/xml"
+	"fmt"
+	"strings"
+
+	"github.com/dustin/go-humanize"
+	"github.com/pkg/errors"
+
+	"github.com/daos-stack/daos/src/control/common/proto/convert"
+	"github.com/daos-stack/daos/src/control/provider/system"
+	"github.com/daos-stack/daos/src/control/server/storage"
+)
+
+// <DimmList>
+//  <Dimm>
+//   <DimmID>0x0001</DimmID>
+//   <Capacity>502.599 GiB</Capacity>
+//   <HealthState>Healthy</HealthState>
+//   <FWVersion>01.00.00.5127</FWVersion>
+//   <PhysicalID>0x001e</PhysicalID>
+//   <DimmUID>8089-a2-1839-000010ce</DimmUID>
+//   <SocketID>0x0000</SocketID>
+//   <MemControllerID>0x0000</MemControllerID>
+//   <ChannelID>0x0000</ChannelID>
+//   <ChannelPos>1</ChannelPos>
+//   <PartNumber>NMA1XXD512GQS</PartNumber>
+//  </Dimm>
+// </DimmList>
+
+type (
+	hexShort    uint32
+	stringSize  uint64
+	stringPlain string
+
+	// DIMM struct represents a PMem AppDirect region.
+	DIMM struct {
+		XMLName          xml.Name    `xml:"Dimm"`
+		ID               hexShort    `xml:"DimmID"`
+		Capacity         stringSize  `xml:"Capacity"`
+		HealthState      stringPlain `xml:"HealthState"`
+		FirmwareRevision stringPlain `xml:"FWVersion"`
+		PhysicalID       hexShort    `xml"PhysicalID"`
```

Comment:
Interestingly this typo doesn't cause an issue with unmarshalling of the XML into native struct because the tag is ignored and the default XML name that is being looked for is the field name (which is the same as the tag we are looking for i.e.  "PhysicalID"). Thanks for pointing this out. The test at https://github.com/daos-stack/daos/pull/13791/files#diff-31357736453e3cb30b544bdc2213f4c06d3f78f6db69f834aea4e8d1887b4cacR170 passes despite typo. Fixed.